catalog/ui/catalog-ui-search-api/src/main/java/org/codice/ddf/catalog/ui/splitter/StorableResource.java

```diff
@@ -11,7 +11,7 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.catalog.ui.metacard.internal;
+package org.codice.ddf.catalog.ui.splitter;
```

Comment:
Since we are moving these interfaces, i think we can remove the export here : https://github.com/codice/ddf/blob/master/catalog/ui/catalog-ui-search/pom.xml#L434